vector/src/main/java/im/vector/riotx/features/reactions/EmojiChooserFragment.kt

```diff
@@ -36,12 +36,10 @@ class EmojiChooserFragment : VectorBaseFragment() {
         viewModel = activity?.run {
             ViewModelProviders.of(this, viewModelFactory).get(EmojiChooserViewModel::class.java)
         } ?: throw Exception("Invalid Activity")
-        viewModel.initWithContect(context!!)
+        viewModel.initWithContext(context!!)
```

Comment:
Don't pass activity context to viewModel. Please use injection to pick what you need from Application context

Reply:
Yes, this fragment need to be refactored completely. Maybe another PR?